conf/modules.config

```diff
@@ -34,6 +34,7 @@ params {
         'sra_fastq_ftp' {
             publish_dir     = 'public_data'
             publish_files   = ['fastq.gz':'', 'md5':'md5']
+            args            = '-C - --max-time 1200'
```

Comment:
Would there be any advantage in setting this to '-C - --max-time 600'?

Reply:
oh sorry - that was a test that got left in, I don't really know what the optimum timeout is.

Reply:
600 (10 mins seems to work fine) but 20 mins would also probs be fine

Reply:
Ok. Let's leave it at 20 mins 👍🏽